tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+oc create namespace kubevirt
+rm -rf _out
+hack/dockerized hack/build-func-tests.sh
+make manifests
+rm -f _out/manifests/testing/cdi-*
+rm -f _out/manifests/testing/kubevirt-config.yaml
+sed -i "s/namespace: kubevirt/namespace: ${KUBEVIRTNAMESPACE}/g" _out/manifests/testing/*.yaml
+_out/tests/tests.test --cdi-namespace=${KUBEVIRTNAMESPACE} \
+	--deploy-testing-infra \
+	--ginkgo.seed=0 \
+	--installed-namespace=${KUBEVIRTNAMESPACE} \
+	--junit-output=${PWD}/xunit_results.xml \
+	--kubeconfig=${KUBECONFIG} \
+	--kubectl-path=${_kubectl} \
+	--oc-path=${_oc} \
+	--path-to-testing-infra-manifests=${PWD}/_out/manifests/testing \
+	--polarion-custom-plannedin=2_3 \
```

Comment:
parameterize maybe please ?

Reply:
What are you talking about? If you understand what this is doing maybe you can share, because I have no clue.

@enp0s3 , do we really need this polarion additions here? I would prefer to remove them.

Reply:
just guess its some version of CNV that say polarion what those tests planned to be fixed on ?
question is if its dynamic, and then would be good to have export an option to update via env var?

Reply:
@EdDev @oshoval All the polarion flags actually [link](https://github.com/kubevirt/kubevirt/blob/master/vendor/kubevirt.io/qe-tools/pkg/ginkgo-reporters/polarion_reporter.go) between the Ginkgo reporter to Polarion. I think they are not mandatory in external runs.